test/e2e/specs/04_userStatus.js

```diff
@@ -1,7 +1,6 @@
 module.exports = {
 
   before: (client) => {
-    client.setWindowPosition(0, 0);
```

Comment:
This is important to keep.

Reply:
On browserstack the browser does not always start at 0, 0

Reply:
I think this was a merge confict :p I will add it back

Reply:
Oops that was me, sorry!

Reply:
@okbel